.github/workflows/test-template.yml

```diff
@@ -762,7 +762,7 @@ jobs:
           timeout_minutes: 10
           max_attempts: 5
           retry_wait_seconds: 120
-          command: docker compose -f docker/docker-compose.yml up --detach postgres mysql mssql mongodb_migrate mongodb_migrate_seed cockroachdb
+          command: docker compose -f docker/docker-compose.yml up --wait --detach postgres mysql mssql mongodb_migrate mongodb_migrate_seed cockroachdb
```

Comment:
`--wait` takes advantage of healthchecks, so with this and the other changes we are sure for instance that `vitess` and `planetscale` are started and fully ready by the time the tests start.